docs/examples/contrib/sqlalchemy/sqlalchemy_declarative_models.py

```diff
@@ -51,6 +59,5 @@ async def get_authors(db_session: "AsyncSession", db_engine: "AsyncEngine") -> l
 
 app = Litestar(
     route_handlers=[get_authors],
-    on_startup=[on_startup],
```

Comment:
this should not be removed if the data is to be seeded (which this PR adds)

Reply:
Indeed, see below!